scopes/react/component-highlighter/component-highlighter/component-highlighter.tsx

```diff
@@ -2,38 +2,51 @@ import React, { useState, useCallback, useEffect } from 'react';
 import classnames from 'classnames';
 import { domToReact, toRootElement } from '@teambit/modules.dom-to-react';
 import { HoverSelector } from '@teambit/ui.hover-selector';
+import { useDebouncedCallback } from 'use-debounce';
 import { Frame } from '../frame';
 import { Label, LabelContainer } from '../label';
-import { isBitComponent } from './bit-react-component';
+import { hasComponentMeta } from './bit-react-component';
 
 import styles from './component-highlighter.module.scss';
 
+type HighlightTarget = {
+  id?: string;
+  element: HTMLElement;
+  /** e.g. 'https://bit.dev/teambit/base-ui/elements/dots-loader', */
+  link?: string;
+  /** e.g. 'https://bit.dev/teambit/base-ui' */
+  scopeLink?: string;
+};
+
 export interface ComponentHighlightProps extends React.HTMLAttributes<HTMLDivElement> {
   disabled?: boolean;
 }
 
 export function ComponentHighlighter({ children, disabled, ...rest }: ComponentHighlightProps) {
-  const [text, setText] = useState<string | undefined>(undefined);
-  const [target, setTarget] = useState<HTMLElement | null>(null);
+  const [target, setTarget] = useState<HighlightTarget | undefined>();
 
-  const handleElement = useCallback((element: HTMLElement | null) => {
+  const _handleElement = useCallback((element: HTMLElement | null) => {
     if (!element) {
-      setTarget(null);
-      setText(undefined);
+      setTarget(undefined);
       return;
     }
 
     const bitComponent = bubbleToBitComponent(element, (elem) => !elem.hasAttribute('data-ignore-component-highlight'));
     if (!bitComponent) return;
 
-    setText(bitComponent.id);
-    setTarget(bitComponent.element);
+    setTarget({
+      element: bitComponent.element,
+      id: bitComponent.id,
+      scopeLink: undefined,
+      link: bitComponent.homepage,
+    });
   }, []);
 
+  const handleElement = useDebouncedCallback(_handleElement, target ? 300 : 0);
```

Comment:
debounce allows people to slide to the bubble without losing the highlighter
```suggestion
  const handleElement = useDebouncedCallback(_handleElement, target ? 300 : 0);
```